src/map/attack.cpp

```diff
@@ -30,7 +30,7 @@
 
 /************************************************************************
  *                                                                      *
- *  Constructor.                                                            *
+ *  Constructor.                                                        *
```

Comment:
If we're in here cleaning up, big block comments that don't provide any additional information (like this one) can go